pkg/rc/farm.go

```diff
@@ -277,8 +278,13 @@ func (rcf *Farm) failsafe(rcFields []rcstore.RCLockResult) {
 		}
 		panic("No RCs are scheduled at all. Create one RC to enable the farm. Panicking to escape a potentially bad situation.")
 	}
+
 	globalReplicaCount := 0
 	for _, rc := range rcFields {
+		if rc.ReplicasDesired == 0 {
```

Comment:
@eh? this looks suspicious. Replica counts can legitimately be zero, right?

Am I regretting asking for `ReplicasDesired` to no longer be a pointer?

I think we only want to panic if we encounter a RawRC whose pointer is nil, not if we see a zero in a cooked RC. I know that we don't have raw RCs when we get to this point, so perhaps that means a change is needed?